cluster/operator-install.sh

```diff
@@ -17,6 +17,13 @@
 set -ex
 
 export VERSION=v0.0.0
+export VM_IMPORT_CONFIG_STATUS=${VM_IMPORT_CONFIG_STATUS:-''}
 
 ./cluster/kubectl.sh create -f _out/vm-import-operator/${VERSION}/operator.yaml
-./cluster/kubectl.sh wait deploy/vm-import-operator -n kubevirt --for=condition=Available --timeout=600s
+./cluster/kubectl.sh create -f _out/vm-import-operator/${VERSION}/vmimportconfig_cr.yaml
```

Comment:
Ondra, could there be a race and teh vm-import-operator deployment will not be ready and line 24 will fail due to that reason since the vm-import-controller will not be created at that time.
We need to add another wait after line 23 - as you did originally
`./cluster/kubectl.sh wait deploy/vm-import-operator -n kubevirt --for=condition=Available --timeout=600s	`
once it is ready - create the vmimportconfig_cr.yaml
and then wait on vm-import-controller

Reply:
There is a problem:
```
deployment.apps/vm-import-operator created
+ ./cluster/kubectl.sh create -f _out/vm-import-operator/v0.0.0/vmimportconfig_cr.yaml
vmimportconfig.v2v.kubevirt.io/vm-import-operator-config created
+ ./cluster/kubectl.sh wait deploy/vm-import-controller -n kubevirt --for=condition=Available --timeout=600s
Error from server (NotFound): deployments.apps "vm-import-controller" not found
```

Reply:
that's why i suggested to add a wait for the vm-import-operator deployment first
then create the vmimport-config cr
and then wait on the vm-import-controller deployment
but there can be a race between the last two - by watching the vmimportconfig status - it should be fine to wait until it is reported to complete it task by making sure the vm-import-controller deployment is ready

Reply:
> Ondra, could there be a race and teh vm-import-operator deployment will not be ready and line 24 will fail due to that reason since the vm-import-controller will not be created at that time.
> We need to add another wait after line 23 - as you did originally
> `./cluster/kubectl.sh wait deploy/vm-import-operator -n kubevirt --for=condition=Available --timeout=600s `
> once it is ready - create the vmimportconfig_cr.yaml
> and then wait on vm-import-controller

Thanks, fixed.



> There is a problem:
> 
> ```
> deployment.apps/vm-import-operator created
> + ./cluster/kubectl.sh create -f _out/vm-import-operator/v0.0.0/vmimportconfig_cr.yaml
> vmimportconfig.v2v.kubevirt.io/vm-import-operator-config created
> + ./cluster/kubectl.sh wait deploy/vm-import-controller -n kubevirt --for=condition=Available --timeout=600s
> Error from server (NotFound): deployments.apps "vm-import-controller" not found
> ```

Thanks, actually it's bug in kubectl, there is no mechanism to wait on non-existent, there is PR to add `--ignore-not-found`, but not yet released. I'll add some WA.